package.json

```diff
@@ -21,12 +21,12 @@
     "dist/**/*"
   ],
   "engines": {
-    "node": ">=12.13.0",
-    "npm": ">=6.12.0"
+    "node": ">=14.21.3",
```

Comment:
These values are based on the [node JS release schedule](https://nodejs.org/en/about/previous-releases), and the latest patch version of node 14 that is available (but also EOL) as well as the npm version bundled with it.